modules/placement-driver/src/main/java/org/apache/ignite/internal/placementdriver/LeaseUpdater.java

```diff
@@ -514,7 +513,9 @@ private void writeNewLease(
         ) {
             HybridTimestamp startTs = clockService.now();
 
-            var expirationTs = new HybridTimestamp(startTs.getPhysical() + longLeaseInterval, 0);
+            long interval = replicationConfiguration.agreementAcceptanceInterval().value();
```

Comment:
I don't know how fast the values can be retrieved from configuration, are you sure that we shouldn't get this only once per iteration in #updateLeaseBatchInternal?